plugins/hls-class-plugin/src/Ide/Plugin/Class/Types.hs

```diff
@@ -45,7 +46,7 @@ data GetInstanceBindTypeSigs = GetInstanceBindTypeSigs
 
 data InstanceBindTypeSig = InstanceBindTypeSig
     { bindName     :: Name
-    , bindRendered :: T.Text
+    , bindRendered :: !T.Text
```

Comment:
Any reason this needs to be strict while the other values arent?

Reply:
It showed up in a profile, the others are likely to already be evaluated I think.

Reply:
Then a comment would be nice.
Also, would a guideline be wise that generally says everything in records needs a bang, and only in certain exceptional cases you don't need them?

Reply:
Yeah, it seems clearer to bang everything in situations like this? Unsure.